src/main/java/ConfirmBuyPropertyUseCase/ConfirmBuyPropertyInteractor.java

```diff
@@ -0,0 +1,97 @@
+package ConfirmBuyPropertyUseCase;
+
+import MainEntities.CampaignAccess;
+import MainEntities.Player;
+import Properties.Property;
+import Tiles.PropertyTile;
+
+public class ConfirmBuyPropertyInteractor implements ConfirmBuyPropertyInputBoundary {
+
+    private final ConfirmBuyPropertyOutputBoundary result;
+    private final Property property;
+    private final Player player;
+    private CampaignAccess campaignAccess;
+
+
+    public ConfirmBuyPropertyInteractor(ConfirmBuyPropertyOutputBoundary buyPropertyOutputBoundary,
```

Comment:
I think it might be better to get the property and player instance variables from the CampaignAccess instead of passing it as a parameter to the constructor.

Reply:
Yes, I agree with you on this point, changed this accordingly.